### PR TITLE
[ENH] Aligned assessment tool workflow UI elements

### LINKED ISSUE #612: The assessment tool workflow is horizontally aligned below the column workflow
At the moment, the annotation workflow uses different layout columns and ends up being split evenly across the viewport. Instead it should be aligned with the column  annotation workflow so that:

~- [ ] the "select tool" dropdown window sits below the "pick category" table, with the same width~
- [x] once a tool is selected, the "here is your tool" table sits below the dropdown, again same width
- [x] The "here are the columns you said were about assessment tools" table should sit below the "column-table", same width

### PR DESCRIPTION
**Pull request by rmanaem**
- Closes #612 

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Tests pass
- [x] Checks pass

For new features:
- [ ] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

**Comment by surchs** [comment]
Shouldn't there be a preview build from netlify here?

File: components/category-select-table.vue [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
     <div>
 
-        <b-row class="no-padding-row">
+        <b-row>
```
Comment (surchs) [inline components/category-select-table.vue:5]: not sure what this does, mind checking again that this was intentional?

**Review by surchs — APPROVED** [review]
:cook: looks good @rmanaem :tada:. I even tried playing with the viewport size and it works great - at least as good as the rest of the app.

File: cypress/e2e/app/simple-e2etest.cy.js
[the remark this answers is not in the document]
Comment (rmanaem) [reply cypress/e2e/app/simple-e2etest.cy.js:1]: Yea good call, I forgot to uncomment before pushing.

File: cypress/support/commands.js [hunk 2]
```diff
@@ -26,16 +26,6 @@
...
-// test setups (i.e. testing across different viewport sizes).
-Cypress.Commands.add("appSetup", () => {
-
-    // Standard setup for annotation tool testing
-
-    // A. Set viewport size
-    // NOTE: Standard 13-inch laptop screen to start, but this can be expanded
-    cy.viewport("macbook-13");
-});
-
```
[the remark this answers is not in the document]
Comment (rmanaem) [reply cypress/support/commands.js:38]: Yea

File: components/category-select-table.vue (hunk 1 above)
Comment (rmanaem) [reply components/category-select-table.vue:5]: I checked when I was working on the layout and there is no built-in `no-padding-row` class in Bootstrap and we don't implement it in the code base either.

**Pull request merged**